release_dashboard/test/widgets/conductor_status_test.dart

```diff
@@ -49,18 +49,22 @@ void main() {
 
     testWidgets('Conductor_status displays correct status with a null state file except a releaseChannel',
         (WidgetTester tester) async {
-      final pb.ConductorState stateIncomplete = pb.ConductorState(
-        releaseChannel: kReleaseChannel,
-      );
+      final pb.ConductorState stateIncomplete = generateConductorState();
```

Comment:
Before the test was setting everything field of the release state to null except the release channel, the scenario is impossible in production (for example, the LUCi dashboard link will never be null) and causing some of the widgets to fail.